wordpress-mobile/WordPress-iOS
# Issue/integrate media picker 0.19

Integrates version 0.19 of media picker.

To test:
 - Check places where the picker is used: Editors, Gravatar image selection, Site icon selection, Featured Image and see if everything is working correctly.
